### ISSUE TITLE
No Bcc Functionality

### ISSUE THREAD

**Issue by aboichev** [Jaben/Papercut#8]
Bcc functionality seems to be missing. 

It would be great to get multiple emails for each bcc recipient for testing purposes. Or alternatively have a label that says something like "this email was also sent to: [list_of_email]. 

Here is a log with multiple RCPT TO which are ignored right now:
``` log
220 myhost
EHLO myhost
250-myhost
250-8BITMIME
250 OK
RSET
250 OK
MAIL FROM: <a@b>
250 <a@b> OK
RCPT TO: <c@d>
250 <c@d> OK
RCPT TO: <e@f>
250 <e@f> OK
RCPT TO: <g@h>
250 <g@h> OK
DATA
354 Start mail input; end with <CRLF>.<CRLF>
.
250 OK
Forcing disconnection from SMTP server.
QUIT
221 Goodbye!
```

**Comment by Jaben** [comment]
Fix in latest 4.5.0.0 release.

**Comment by jrb0001** [comment]
Your fix caused another issue: [Sending MailMessage with To Recipient having a DisplayName Generates Erroneous Bcc](https://papercut.codeplex.com/workitem/9884)
